app/assets/stylesheets/modules/_service-standard-point.scss

```diff
@@ -1,45 +1,41 @@
-.service-standard-point {
+.app-c-service-standard-point {
+  $gutter-tablet: (govuk-spacing(6) * 1.5);  // 45px
 
-  $gutter-tablet: ($gutter * 1.5);  // 45px
-
-  border-top: 1px solid $border-colour;
-  margin-top: 1.5em;
-  padding-top: 1.5em;
+  border-top: 1px solid $govuk-border-colour;
+  margin-top: govuk-spacing(5);
+  padding-top: govuk-spacing(5);
 
   &:first-child {
-    margin-top: 0;
+    margin-top: govuk-spacing(0);
   }
 
   &__title {
-    @include bold-24;
-    margin-bottom: 0.5em;
-    margin-left: $gutter;
+    margin-left: govuk-spacing(6);
 
-    @include media(tablet) {
+    @include govuk-media-query($from: tablet) {
       margin-left: $gutter-tablet;
     }
   }
 
   &__number {
     float: left;
-    margin-left: -$gutter;
+    margin-left: (0 - govuk-spacing(6));
 
-    @include media(tablet) {
+    @include govuk-media-query($from: tablet) {
       margin-left: -$gutter-tablet;
```

Comment:
will it still exist with frontend_toolkit removal?

Reply:
Yes, as `$gutter-tablet` is set within the [`_service-standard-point.scss` file](https://github.com/alphagov/service-manual-frontend/pull/518/files/875fb6963486a92e1d2ea6fcacbb4990c3d1a00c#diff-a2d4e3b902bbce25f8bb079c6be9473bR2) which is :

```
$gutter-tablet: (govuk-spacing(6) * 1.5);  // 45px
```